tigerpath/majors_and_certificates/majors/GEO_2018.json

```
@@ -0,0 +1,467 @@
+                    "GEO 363",
+                    "GEO 417"
+                  ]
+                },
+                {
+                  "name": "Electives",
+                  "max_counted": 1,
+                  "min_needed": 5,
+                  "explanation": "Five elective 300/400 level geosciences courses.",
+                  "course_list": [
+                    "GEO 3*",
+                    "GEO 4*"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Environmental Geochemistry",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "General Requirements",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "req_list": [
+                {
+                  "name": "Statistics",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "One statistics requirement, to be completed by the end of the junior year (GEO 422 or ORG 245).",
+                  "completed_by_semester": 6,
+                  "course_list": [
+                    "GEO 422",
+                    "ORF 245"
+                  ]
+                },
+                {
+                  "name": "Math and Science",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": "Four required core math and science courses that vary depending on the chosen specialization. ISC 231-234 and/or AP credit may substitute for some of these courses. Students interested in graduate school are encouraged to take more than these minimum basic science requirements.",
+                  "course_list": [
+                    "PHY 103",
+                    "MOL 214",
+                    "CHM 201",
+                    "CHM 202"
+                  ]
+                },
+              ]
+            },
+            {
+              "name": "Departmental Requirements",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "req_list": [
+                {
+                  "name": "Core Geoscience",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": "Two core geoscience requirements that vary depending on the chosen specialization.",
+                  "course_list": [
+                    "GEO 363",
+                    "GEO 360"
+                  ]
+                },
+                {
+                  "name": "Electives",
+                  "max_counted": 1,
+                  "min_needed": 5,
+                  "explanation": "Five elective 300/400 level geosciences courses.",
+                  "course_list": [
+                    "GEO 3*",
+                    "GEO 4*"
+                  ]

```

Same here. Not sure what "frs" means here in this context though.
```suggestion
                  "course_list": [
                    "GEO 361",
                    "GEO 370",
                    "GEO 417",
                    "GEO 418",
                    "GEO 428",
                    "GEO 470",
                    "CHM 301",
                    "CEE 311",
                    "CEE 306",
                    "CEE 307",
                    "CEE 471"
                  ]
```

---

FRS is freshman seminar, which also made me think the electives were an example set of courses.